pkg/scheduler/framework/plugins/nodename/node_name.go

```diff
@@ -27,6 +27,8 @@ import (
 )
 
 // NodeName is a plugin that checks if a pod spec node name matches the current node.
+// This plugin is enabled by default but is currently not functioning.
+// It remains here as an example of how to implement Filter plugin (#114396).
```

Comment:
```suggestion
// This plugin is enabled by default but is not functioning.
// It remains here as an example of how to implement Filter plugin,
// and we cannot remove it because it's not allowed to change the default in APIs (#114396).
```